src/api/XRDevicePose.js

```diff
@@ -14,7 +14,7 @@
  */
 
 const PRIVATE = Symbol('@@webxr-polyfill/XRDevicePose');
-import { mat4_identity } from '../math';
+import * as mat4 from '../../node_modules/gl-matrix/src/gl-matrix/mat4';
```

Comment:
Does `gl-matrix/src/gl-matrix/mat4` work? I think that's preferable to the relative node_modules path